src/Psalm/Internal/FileManipulation/FunctionDocblockManipulator.php

```diff
@@ -332,7 +334,29 @@ private function getDocblock(): string
                 foreach ($parsed_docblock->tags['param'] as &$param_block) {
                     $doc_parts = CommentAnalyzer::splitDocLine($param_block);
 
+                    // If there's no type
+                    if (($doc_parts[0] ?? null) === '$' . $param_name) {
+                        // If the parameter has a description add that back
+                        if (count($doc_parts) > 1) {
+                            $new_param_block .= " ". implode(" ", array_slice($doc_parts, 1));
+                        }
+
+                        if ($param_block !== $new_param_block) {
+                            $modified_docblock = true;
+                        }
+
+                        $param_block = $new_param_block;
+                        $found_in_params = true;
+                        break;
+                    }
+
+                    // If there is a type
                     if (($doc_parts[1] ?? null) === '$' . $param_name) {
```

Comment:
Doesn't it actually handle the case when there's *no* type?

Reply:
Ah, ignore me. `$doc_parts[0]` is (normally) a type, and `$doc_parts[1]` is the param name.